### PR TITLE
Polish workflows

### PR DESCRIPTION
**Pull request by masayag**
Closes https://github.com/parodos-dev/serverless-workflows-helm/issues/41

### CODE REVIEW

File: charts/workflows/charts/greeting/README.md
Comment (gabriel-farache) [inline charts/workflows/charts/greeting/README.md:1]: why did you remove the content?

File: charts/workflows/charts/greeting/values.schema.json
Comment (gabriel-farache) [inline charts/workflows/charts/greeting/values.schema.json:1]: why updating generated file?

File: charts/workflows/charts/greeting/README.md
Comment (masayag) [reply charts/workflows/charts/greeting/README.md:1]: By default, too much is being auto-generated by the tool from the READme.
There is not input required for the greeting workflow.

File: charts/workflows/charts/greeting/values.schema.json
Comment (masayag) [reply charts/workflows/charts/greeting/values.schema.json:1]: seems the generated schema by the tool in the README failed to be opened in the Helm view in developer perspective on OCP

File: charts/workflows/charts/greeting/README.md
Comment (gabriel-farache) [reply charts/workflows/charts/greeting/README.md:1]: oki doki
